src/staticsampler.jl

```diff
@@ -96,8 +100,10 @@ function Nested(ndims,
 end
 
 default_update_interval(p::Proposals.Uniform) = 1.5
-default_update_interval(p::Proposals.RWalk) = 0.15p.walks
-default_update_interval(p::Proposals.RStagger) = 0.15p.walks
+default_update_interval(p::Proposals.RWalk) = 0.15 * p.walks
+default_update_interval(p::Proposals.RStagger) = 0.15 * p.walks
+default_update_interval(p::Proposals.Slice) = 0.9 * ndims * p.slices
```

Comment:
this code won't work- `ndims` isn't defined anywhere. What we'll have to do is update all of these-

```suggestion
default_update_interval(p::Proposals.RWalk, ndims) = 0.15 * p.walks
default_update_interval(p::Proposals.RStagger, ndims) = 0.15 * p.walks
default_update_interval(p::Proposals.Slice, ndims) = 0.9 * ndims * p.slices
```

you'll have to also change the usage in the `Nested` function and update the tests.

Reply:
okay

Reply:
yup